tests/general/default_config_test.go

```diff
@@ -35,8 +35,9 @@ func TestDefaultGatewayConfig(t *testing.T) {
 		"",
 		func(collector testutils.Collector) testutils.Collector {
 			return collector.WithArgs().WithEnv(map[string]string{
-				"SPLUNK_ACCESS_TOKEN": "not.real",
-				"SPLUNK_REALM":        "not.real",
+				"SPLUNK_LISTEN_INTERFACE": "0.0.0.0",
```

Comment:
To vet the default behavior it's probably best to remove this and if you'd like add new tests to confirm the env var is picked up where expected.